### PR TITLE
updated role and cleanup image

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
- Made `cleanupCRD` deprecated in a favour of `crd.cleanup.enabled`
- Made `cleanupImage` deprecated in a favour of `crd.cleanup.image`
- Made `watchNamespace` string deprecated in a favour of `watchNamespaces` slice

### CODE REVIEW

File: charts/victoria-metrics-operator/CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 ## Next release
 
-- TODO
+- Renamed `cleanupCRD` to `cleanupCRD.enabled`
+- Renamed `cleanupImage` to `cleanupCRD.image`
+- Replaced `watchNamespace` string with `watchNamespaces` slice
```
Comment (f41gh7) [inline charts/victoria-metrics-operator/CHANGELOG.md:7]: People hate such changes. We must have strong motivation to introduce it.

File: charts/victoria-metrics-operator/CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
...
+- Removed unnecessary Role and RoleBinding
```
Comment (f41gh7) [inline charts/victoria-metrics-operator/CHANGELOG.md:8]: It'd be great to have a guide or example with minimal operator RBAC permissions for single/multi namespace deployment.

Comment (f41gh7) [reply charts/victoria-metrics-operator/CHANGELOG.md:8]: Also, operator is always requires single namespace permissions for leader-election mechanism.

File: charts/victoria-metrics-operator/CHANGELOG.md (hunk 1 above)
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-operator/CHANGELOG.md]: updated changelog and added support of both old and new properties, marked old ones as deprecated

File: charts/victoria-metrics-operator/CHANGELOG.md (hunk 2 above)
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-operator/CHANGELOG.md]: reverted role, but left permissions for leader election only

**Review by zekker6 — APPROVED** [review]

**Review by f41gh7 — APPROVED** [review]
LGTM

**Pull request merged**